doc/release-notes/release-7.sgml.in

```diff
@@ -57,6 +58,23 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 </descrip>
 
+<sect1>Removed purge tool
+<p>The <em>purge</em> tool (also known as <em>squidpurge</em>, and <em>squid-purge</em>)
+   was limited to managing UFS/AUFS/DiskD caches.
+
+<p>The cache contents display and search it provided can be obtained with a script
+   searching the cache manager <em>objects</em> report.
+
+<p>This tool used the custom <em>PURGE</em> HTTP method to remove cache
+   objects. This can be performed directly on any Squid configured to allow
+   the method. Like so:
+<verb>
+    acl PURGE method PURGE
+    http_access allow manager PURGE
+</verb>
```

Comment:
Is not that configuration required for the `purge` tool to work? If so, feels odd to suggest that folks should _add_  this configuration if they are thinking about replacing `purge` with another program that sends PURGE requests.
```suggestion
<p>This tool used custom HTTP <em>PURGE</em> request method to remove cache
   objects. PURGE requests can be sent using popular clients like curl. For example:
<verb>
    $ curl --proxy ... --request PURGE http://example.com/url-to-purge
</verb>
```